lib/logger/bindLogger.js

```diff
@@ -0,0 +1,66 @@
+const { performance } = require('perf_hooks');
+const { getDatadogSecret } = require('./getDatadogSecret');
+const { initLogger } = require('./logger');
+const { initDataDogMetricsApi } = require('./metricsApiWrapper');
+
+const initDependencies = async () => {
+	const logVersion = 'v1';
+	if (process.env.FALAFEL_ENABLE_LOGGING !== 'true') {
+		return { metricsApiWrapper: undefined, logger: undefined };
+	}
+	try {
+		const apiKey = await getDatadogSecret(process.env.DATADOG_SSM_PARAM_KEY, process.env.AWS_REGION);
```

Comment:
personally I would prefer not to pass ENVs as variables unless there are cases where we would have to invoke the function with different ENVs.